featuretools/primitives/options_utils.py

```diff
@@ -0,0 +1,228 @@
+import warnings
+
+from featuretools import primitives
+from featuretools.feature_base import IdentityFeature
+
+
+def _get_primitive_options():
+    # all possible option keys: function that verifies value type
+    return {'ignore_entities': list_entity_check,
+            'include_entities': list_entity_check,
+            'ignore_variables': dict_to_list_variable_check,
+            'include_variables': dict_to_list_variable_check,
+            'ignore_groupby_entities': list_entity_check,
+            'include_groupby_entities': list_entity_check,
+            'ignore_groupby_variables': dict_to_list_variable_check,
+            'include_groupby_variables': dict_to_list_variable_check}
+
+
+def dict_to_list_variable_check(option, es):
+    if not (isinstance(option, dict) and
+            all([isinstance(option_val, list) for option_val in option.values()])):
+        return False
+    else:
+        for entity, variables in option.items():
+            if entity not in es:
+                warnings.warn("Entity \'%s\' not in entityset" % (entity))
```

Comment:
do you need to use the the escape `\`here? doesn't using double quotes on the outside and singel quotes on the side get around that?